docs/fx-core/using-existing-aad.md

```diff
@@ -95,7 +91,7 @@ This doc is for using existing Azure AD app or Manually Create Azure AD app for
        "objectId": **Object ID**,
        "clientId": **Application (client) ID**,
        "clientSecret": {{ $env.AAD_APP_CLIENT_SECRET }},
-       "accessAsUserScopeId": **Access As User Scope ID**
+       (optional) "accessAsUserScopeId": **Access As User Scope ID**
```

Comment:
Todo: Also need to update src\schemas\envConfig.json file to make 'accessAsUserScopeId' optional